charts/redis/templates/redis-pvc.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.managed_redis.disabled }}
```

Comment:
Can we rename this file to `PersistentVolumeClaim.yaml` ?

Reply:
Yup, It's updated